### DISCUSSION TITLE
Will true public teams come back one day?

### DISCUSSION THREAD

**Discussion by typed-sigterm** [community/community#125802] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

> People who are not members of the organization cannot view any teams.

Will the **true** public teams come back one day after [the email storm event](https://github.com/EpicGames/Signup/pull/24)?

For some open-source projects, it is really important to know which teams and who people are in the team. I think providing an option to publicly disclose a team member list to everyone should not lead to such incidents happening again.

**Comment by Abhishek22K** [comment]
There's a compelling argument for platforms to create solutions that provide open-source projects the transparency they want, even though truly public teams might not reappear in their original form. It is possible to satisfy the needs of open-source projects without rehashing old problems by striking a balance between transparency, privacy, and security, and including the community in these decisions. To remain updated and help with the development of these features, follow GitHub updates and participate in community discussions.

**Comment by typed-sigterm** [comment]
#134957
